recipes/chemfiles-python/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "913a9ca911287e342918458619c29dfd094f4b25f7d6866048b9968fc63be207" %}
+
+package:
+  name: chemfiles-python
+  version: {{ version }}
+
+about:
+  home: http://chemfiles.github.io
+  license: MPL-v2.0
+  summary: Python binding to chemfiles, a modern library for chemistry file reading and writing
+  doc_url: http://chemfiles.github.io/chemfiles.py/latest/
+  dev_url: https://github.com/chemfiles/chemfiles.py
+
+build:
+  script:  # [not win]
+      - export MACOSX_DEPLOYMENT_TARGET=""  # [osx]
+      - python setup.py install # [not win]
```

Comment:
Please drop the selector and add add the arguments ` --single-version-externally-managed --record=record.txt`.